src/index.html

```diff
@@ -66,30 +67,127 @@
   "exportViewUrl": "/api/v1/viewconfs",
   "views": [
     {
-      "initialXDomain": [
-        1069795.099783097,
-        1070508.6726894341
-      ],
+      // Example Pete
+      //"initialXDomain": [1069795,1070508],
+
+      // Variant + Soft Clipping (left)
+      "initialXDomain": [1558492965,1558493030],
+
+      // Insertions + Soft Clipping both sides
+      //"initialXDomain": [105706,105906],
+
+      // Hard Clipping (left and both sides)
+      //"initialXDomain": [145009,145409],
+
+      // Deletions
+      //"initialXDomain": [128870,129070],
+
       "initialYDomain": [
         180520.26598912096,
         180863.4346387336
       ],
       "tracks": {
         "top": [
+          // {
+          //   "type": "pileup",
+          //   "uid": "WBTUIwm0SB2gcas7wzwztA",
+          //   "tilesetUid": "NLxWZTCoSzei1lkH4yVJmg",
+          //   "server": "https://resgen.io/api/v1",
+          //   "options": {
+          //     "axisPositionHorizontal": "right",
+          //     "axisLabelFormatting": "normal",
+          //     "outlineReadOnHover": false,
+          //     "showMousePosition": false,
+          //     "name": "HG002 Illumina (60x)",
+          //     "labelPosition": "bottomLeft",
+          //     "groupBy": null,
+          //     "colorScale": [
+          //       "#08519c",
+          //       "#6baed6",
+          //       "#993404",
+          //       "#fe9929",
+          //       "#808080",
+          //       "#DCDCDC"
+          //     ]
+          //   },
+          //   "width": 651,
+          //   "height": 362
+          // },
+          {
+            "uid": "AdlJsUYFRzuJRZyYeKDX2A",
+            "type": "chromosome-labels",
+            "width": 811,
+            "height": 30,
+            "server": "//higlass.io/api/v1",
+            "options": {
+              "color": "#808080",
+              "stroke": "#ffffff",
+              "fontSize": 12,
+              "fontIsLeftAligned": false,
+              "showMousePosition": false,
+              "mousePositionColor": "#000000"
+            },
+            "filetype": "chromsizes-tsv",
+            "tilesetUid": "NyITQvZsS_mOFNlz5C2LJg"
+          },
           {
-            "type": "pileup",
-            "uid": "WBTUIwm0SB2gcas7wzwztA",
-            "tilesetUid": "NLxWZTCoSzei1lkH4yVJmg",
-            "server": "https://resgen.io/api/v1",
+            "uid": "fastaex",
+            "data": {
+              "type": "fasta",
+              "faiUrl": "https://aveit.s3.amazonaws.com/higlass/data/sequence/hg38.fa.fai",
+              "fastaUrl": "https://aveit.s3.amazonaws.com/higlass/data/sequence/hg38.fa",
+              "chromSizesUrl": "https://aveit.s3.amazonaws.com/higlass/data/sequence/hg38.mod.chrom.sizes"
+            },
+            "type": "horizontal-sequence",
+            "width": 768,
+            "height": 25,
+            "server": "https://cgap-higlass.com/api/v1",
```

Comment:
Not related to this PR, but why does the sequence track require a server if urls are provided in the `data` section?

Reply:
I doesn't need it. I removed it.